aptible/resource_endpoint_test.go

```diff
@@ -313,6 +313,14 @@ func TestAccResourceEndpoint_expectError(t *testing.T) {
 				Config:      testAccAptibleEndpointInvalidMultipleContainerPortFields(),
 				ExpectError: regexp.MustCompile(`(?i)do not specify container ports AND container port`),
 			},
+			{
+				Config:      testAccAptibleEndpointInvalidSharedWithNoDomain(),
+				ExpectError: regexp.MustCompile(`(?i)must specify a domain`),
+			},
+			{
+				Config:      testAccAptibleEndpointInvalidSharedWithWildcardDomain(),
+				ExpectError: regexp.MustCompile(`(?i)cannot use domain`),
+			},
 		},
```

Comment:
Do you want to add a test(s) for switching to/from shared?

Reply:
Done. Tested that it works by running against my sandbox.